src/gridstack.js

```diff
@@ -1092,8 +1093,9 @@
       Utils.removeStylesheet(this._stylesId);
     }
     this._stylesId = 'gridstack-style-' + (Math.random() * 100000).toFixed();
-    // insert style to parent (instead of 'head') to support WebComponent
-    this._styles = Utils.createStylesheet(this._stylesId, this.el.parentNode);
+    var styleLocation = this.opts.styleInHead ? false : this.el.parentNode
```

Comment:
I would use null or undefined instead since parent is an element pointer (issue in typescript version) but works here in JS.

Reply:
changed to `undefined`